go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/btcsuite/goleveldb v1.0.0 // indirect
 	github.com/coreos/bbolt v0.0.0-20180223184059-7ee3ded59d4835e10f3e7d0f7603c42aa5e83820
 	github.com/davecgh/go-spew v1.1.1
+	github.com/fortytw2/leaktest v1.3.0
```

Comment:
Why's this needed? Would rather use external tools that add another dep.

Reply:
Not entirely convinced, because I don't see the practical downside of having this dependency in the project and only using it in tests that run fully without blocking. For those test parallel wouldn't be a significant speed up. But removed it anyway.